cypress/integration/landing_page_spec.js

```diff
@@ -1,5 +1,83 @@
-describe('Shared Milk user flows', () => {
-  it('Should confirm that true is equal to true', () => {
-    expect(true).to.equal(true)
+describe('Shared Milk landing page user flow', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
   });
-});
\ No newline at end of file
+
+  it('should display page title and navigation links on page load', () => {
+    cy.get('h1').contains('No Use Cryin\' Over Shared Milk')
```

Comment:
@cagallo good work building out the nav bar! 
Let's build out the footer tests too.

Reply:
On that!